Sources/TuistKit/Mappers/Factories/GraphMapperFactory.swift

```diff
@@ -50,6 +51,8 @@ public final class GraphMapperFactory: GraphMapperFactorying {
     func `default`() -> [GraphMapping] {
         var mappers: [GraphMapping] = []
         mappers.append(UpdateWorkspaceProjectsGraphMapper())
+        mappers.append(PruneOrphanExternalTargetsGraphMapper())
+        mappers.append(ExternalProjectsPlatformNarrowerGraphMapper())
```

Comment:
This might not immediately propagate to `tuist-cloud` as we have our custom `GraphMapperFactory`. Should be relatively easy to reuse the definition from here (we already do it for some other mapper methods)